run/src/js/Seeker.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Box from "./Box";
+
+const URL_JSON = "https://api.myjson.com/bins/uptto";
+var agentData = {};
+class Seeker extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {}
+    };
+    this.updateState = this.updateState.bind(this);
+  }
+  updateState(e) {
```

Comment:
you can define `updateState = (e) => { ` so you don't have to use this line in the constructor `this.updateState = this.updateState.bind(this);`